cluster/gce/gci/configure-helper.sh

```diff
@@ -1276,7 +1277,7 @@ clusters:
 - name: local
   cluster:
     insecure-skip-tls-verify: true
-    server: https://localhost:443
+    server: https://${KUBERNETES_MASTER_NAME:-localhost}:443
```

Comment:
should the change in behavior be configured using a env param? I am worried about this breaking and us not having a fallback.

Reply:
Added feature flag.